README.md

```diff
@@ -45,6 +43,18 @@ Today the core uses of HELICS are in the energy domain, where there is extensive
 
 Previous and existing use cases have stretched across a wide range of scales in time and spatial area, from transient dynamics to long-term planning studies, and from individual appliance behavior to nation-wide simulations.
 
+## Philosophy of HELICS
+
+The design and development of HELICS is driven by a number of philosophical considerations that have a clear path to design decisions in the code
+
+- Make it as easy as possible for federates of all kinds to work together
+- Federates cannot impose restrictions or requirements on other federates
+- Federates should maintain control and autonomy
+- The design should be layered and modular to be adaptable to a wide variety of circumstances
+- Centralized control should be minimized
+
+These design priorities directed much of the design of HELICS and supporting tools, including operation as a library vs RTI, the use of distributed timing and control, and giving federates fine grained control over the time management and operations. These support an underlying belief driving co-simulation that "Simulations are Better Together"
```

Comment:
Who is the target audience for this? It feels like it almost fits in with the motivation section more than something leading into getting started.

Reply:
I made it a subheading of the introduction, though I could see moving it to the motivation section as well.

Reply:
I think the mention of use cases added to the first sentence helps make it feel like it is for a slightly larger audience and makes it tie in better with the preceding introduction.